xarray/core/duck_array_ops.py

```diff
@@ -378,9 +332,7 @@ def f(values, axis=None, skipna=None, **kwargs):
 std.numeric_only = True
 var = _create_nan_agg_method("var")
 var.numeric_only = True
-median = _create_nan_agg_method(
-    "median", dask_module=dask_array_compat, invariant_0d=True
```

Comment:
delete median from `dask_array_compat`?

Reply:
Are you asking if this was deliberate? The line is just replaced with `median = _create_nan_agg_method("median", invariant_0d=True)`, so all this does is removed the possible delegating to dask.

There is no median function in `dask_array_compat.py`.

Reply:
Oh weird. The old line looked like it was importing median from dask_array_compat.

Sorry for the noise

Reply:
No worries!

If you wanna help me work out why some of the test CI runs fail whilst my tests pass locally that would be helpful :grin:

Reply:
do you have iris installed?

This looks like an iris version thing:
```
# If the Iris version supports it then we should have a dask array
        # at each stage of the conversion
        if hasattr(actual, "core_data"):
>           assert isinstance(original.data, type(actual.core_data()))
E           AssertionError: assert False
```

Reply:
see the comment on `masked_invalid`